dtests/src/resources/regression_docs/functionalTestList.md

```diff
@@ -0,0 +1,45 @@
+#### This is the list of Hydra tests that should be run for a full regression for Snappy.
+
+## Functional tests to be run in regressions
+
+Following is the list of tests run in snappy regression. Any new bt added and needs to be run must be added here and in the [snappyRegressionScript.sh](https://github.com/SnappyDataInc/snappydata/tree/master/dtests/src/test/java/io/snappydata/hydra/snappyRegressionScript.sh)
+
+1. sample.bt
+
+2. distJoin.bt
+
+3. clusterRestartWithPersistentRecovery.bt
+    *Test upgradtion from gemxd cluster to snappy cluster. Modify and use [local.smartConnectorMode.conf](https://github.com/SnappyDataInc/snappydata/tree/master/dtests/src/test/java/io/snappydata/hydra/local.smartConnectorMode.conf)*
```

Comment:
The description needs to be corrected here. Please refer clusterRestartWithPersistentRecovery.conf for the correct test description.